bundles/test/org.eclipse.smarthome.magic/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,10 @@ Import-Package:
  com.google.common.collect,
  javax.measure.quantity,
  javax.servlet.http,
+ org.apache.log4j,

```

Comment:
Why?

Reply:
Mistake by using the wrong Eclipse recommendation. Thanks for noticing it!